test/src/test/java/org/corfudb/runtime/view/ObjectsViewTest.java

```diff
@@ -122,6 +123,25 @@ public void abortedTransactionDoesNotConflict()
         assertThat((String) args[1]).isEqualTo("v2");
     }
 
+    @Test
+    public void incorrectNestingTest() {
+        CorfuRuntime r1 = getDefaultRuntime();
+        CorfuRuntime r2 = CorfuRuntime.fromParameters(CorfuRuntime.CorfuRuntimeParameters.builder()
+                .nettyEventLoop(NETTY_EVENT_LOOP)
+                .build());
+        SMRMap<String, String> map = getDefaultRuntime().getObjectsView()
```

Comment:
If we use the already declared r1 here (L132) to call the getObjectsView, the code would be more expressive.